ui/de-lib/src/main/java/org/iplantc/de/theme/base/client/desktop/BaseDesktopAppearance.java

```diff
@@ -63,6 +64,10 @@
 
         @Source("org/iplantc/de/theme/base/client/desktop/user.png")
         ImageResource userPrefImg();
+
+        @DataResource.MimeType("font/opentype")
+        @Source("org/iplantc/de/theme/base/client/desktop/Texta_Font/Texta-Bold.otf")
+        DataResource textaBold();
```

Comment:
Operating as usual off of no experience, why does this need to be here when both of the font variants are in `IplantBaseWindowAppearance.java` below? I recall one of the other times I asked this sort of thing it was a Belphegor vs. DE difference but it doesn't look like it here.